nixos/modules/services/misc/gitea.nix

```diff
@@ -159,7 +159,8 @@ in
 
         socket = mkOption {
           type = types.nullOr types.path;
-          default = null;
+          default = if (cfg.database.createDatabase && usePostgresql) then "/run/postgresql" else if (cfg.database.createDatabase && useMysql) then "/run/mysqld/mysqld.sock" else null;
+          defaultText = "null";
```

Comment:
Isn't this a bit misleading when generating docs for a system with gitea and postgresql?

Reply:
The default database type is `sqlite` so the default value for `socket` will be `null`.